settings.gradle

```diff
@@ -99,7 +99,7 @@ includeWithFlags ':eureka',                              'java', 'publish', 'rel
 includeWithFlags ':grpc',                                'java', 'publish', 'relocate', 'native'
 includeWithFlags ':grpc-kotlin',                         'java', 'publish', 'relocate', 'kotlin-grpc', 'kotlin', 'native'
 includeWithFlags ':grpc-protocol',                       'java', 'publish', 'relocate', 'native'
-includeWithFlags ':graphql',                             'java', 'publish', 'relocate', 'native'
+includeWithFlags ':graphql',                             'java11', 'publish', 'relocate', 'native'
```

Comment:
Do you mind leaving an explanation on why this is done?
In case there are any users out there using `graphql` with java8 ask about this, I would like to have an explanation on why this was necessary.